From f288367c758de16fc013c6f4c9851ca85813eb59 Mon Sep 17 00:00:00 2001
From: Erik Johansson <erijo@licq.org>
Date: Thu, 31 Oct 2013 21:05:42 +0100
Subject: [PATCH] Fix build error

Must include QObject to get the definition for Q_OBJECT. But according to the
documentation, Q_OBJECT should only be used for objects derived from QObject so
use Q_GADGET instead. Don't really know why it is needed at all, but this
builds at least.
---
 qt4-gui/src/helpers/usercodec.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/helpers/usercodec.h b/src/helpers/usercodec.h
index eb1df23..74f87e8 100644
--- a/src/helpers/usercodec.h
+++ b/src/helpers/usercodec.h
@@ -20,13 +20,14 @@
 #ifndef USERCODEC_H
 #define USERCODEC_H
 
+#include <QObject>
 #include <QString>
 
 namespace LicqQtGui
 {
 class UserCodec
 {
-  Q_OBJECT
+  Q_GADGET
 
 public:
   struct encoding_t
-- 
1.8.5.1

